images/fcos-bastion-image/root/usr/lib/systemd/system/tang.service

```
@@ -0,0 +1,21 @@
+[Unit]
+Description=Creates the TANG service
+Wants=network-online.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ConditionDirectoryNotEmpty=/var/opt/tang/
```

not 100% sure, but this could be better to notify the unit is failing:

```suggestion
AssertDirectoryNotEmpty=/var/opt/tang/
```

Also, should this go in the unit section?

---

Based on systemd.unit man page...
~~~
Before the unit is started, systemd will verify that the specified conditions and asserts are true. If not, the starting of the unit will be (mostly silently) skipped (in case of conditions), or aborted with an error message (in case of asserts).
~~~
You are right, the `Assert` is more appropriate, and yes, it should go into the Unit section, let me fix that, thks.